test/test_checks.py

```diff
@@ -33,11 +33,10 @@ def test_stdin(self):
 
         stdout_lines = stdout.splitlines()
         self.assertEqual(stderr, b'')
-        self.assertEqual(stdout_lines, [
-            b'stdin:1:25: Q000 Remove bad quotes.',
-            b'stdin:2:25: Q000 Remove bad quotes.',
-            b'stdin:3:25: Q000 Remove bad quotes.',
-        ])
+        self.assertEqual(len(stdout_lines), 3)
```

Comment:
Right, we want to eagerly fail if we have too many lines. Additionally, an `IndexError` might exist if we have too low of a count and do this at the end. In my experience, this is the best setup